src/client/components/MealList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+
+const MealList = () => {
+  const [meal, setMeal] = useState();
```

Comment:
A more accurate naming would be essential.
It is an array of meals, so meals and setMeals 🙏🏽